src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -61,11 +61,13 @@ const populateUISchemaForDisplay = (uiSchema, ionField) => {
   if (display.inputType === 'radio') {
     uiSchema.options = display.options;
   } else if (display.inputType === 'select') {
-    uiSchema.wide = true;
-    //it will create a placeholder for dropdowns, by default it will show 'Select an Option'
-    uiSchema.options = Object.assign({'': ''}, ionOptionsToUiOptions(display.options));
-  } else if (display.inputType === 'select_country_code') {
-    Object.assign(uiSchema, countryUISchema);
+    if(!_.isEmpty(display.format) && display.format === 'country-code'){
```

Comment:
what's special about `countryUISchema` ? it still does have same `type` and `wide` attributes. The only change is `options` and `value`..

i think it's time we can eliminate `countryUISchema` and just update `options` and `value`

Reply:
updated based on the suggestion. Thank you!